src/components/BoardHeader/HeaderMenu/HeaderMenu.tsx

```diff
@@ -112,6 +115,25 @@ const HeaderMenu = (props: HeaderMenuProps) => {
             <label className="item-button__label">{state.board!.showNotesOfOtherUsers ? "Hide" : "Show"} notes of other users</label>
           </button>
         </li>
+        <li className="header-menu__item">
+          <button
+            className="menu__item-button"
+            onClick={() => {
+              store.dispatch(ActionFactory.editUserConfiguration(Parse.User.current()?.id || "", {showHiddenColumns: !state.user?.showHiddenColumns}));
```

Comment:
As we said

Btw: `Parse.User.current()?.id ?? ""` would be the more beautiful alternative to your solution